### PR TITLE
[MIG] stock_putaway_by_route: Migration to 14.0

### PR DESCRIPTION
**Pull request by dzungtran89**
Dependencies:
- stock_putaway_hook: https://github.com/OCA/stock-logistics-workflow/pull/772

### CODE REVIEW

**Review by mmequignon — APPROVED** [review]
LG

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+odoo14-addon-stock-putaway-hook @git+https://github.com/akretion/stock-logistics-workflow@14.0-mig-stock_putaway_hook#subdirectory=setup/stock_putaway_hook
```
Comment (sebalix) [inline test-requirements.txt]: The module has now been merged, to remove.

**Comment by dzungtran89** [comment]
@sebalix , the code has been updated

**Review by sebalix — APPROVED** [review]

**Pull request merged**